src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,18 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+interface TodoInfoProps {
```

Comment:
Just use `Props` for the props types name
```suggestion
interface Props {
```

Reply:
Fix in all cases